sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -2180,6 +2183,96 @@ SUBROUTINE MDLFLD
                endif
             END IF
            ENDIF
+
+!===============
+! AQF
+!===============
+
+        !---- O3 ----
+
+           IF (IGET(267)>0) THEN
```

Comment:
@hsinmulin-NOAA The ID 267 for output O3 is already available in this routine.  May I know the reason for adding in AQF session?

Reply:
@hsinmulin-NOAA I would think you don't need another block of "IF(IGET(267)>0) THEN" since this block is available in the current UPP.

Reply:
@hsinmulin-NOAA In UPP code read "o3mr"(ozone mixing) from fv3 based model history files in netcdf format, 
 assign value to an UPP interval variable "o3" and write out in GRIB2 with name "O3MR". So variable o3 is just an interval variable.

Reply:
@hsinmulin-NOAA There is no reason you read the same variable twice in the read interface.